### ISSUE TITLE
Small fonts throughout the website - particularly in the mobile menu 

### ISSUE THREAD

**Issue by CarelleR** [CarelleR/ProgrammingProjectOUA_2023_SP3_Team2#121]
![387329610_1542825636555241_5315952319955544922_n](https://github.com/CarelleR/ProgrammingProjectOUA_2023_SP3_Team2/assets/71162432/197e2a3b-7abb-48b5-93c4-de1a039d45f6)

Very small menu text for mobile menu.

We should review all text on the website. It should be at least 16 px minimum.

https://www.learnui.design/blog/mobile-desktop-website-font-size-guidelines.html#:~:text=This%20is%20important.,size%20of%20at%20least%2016px.

https://www.smashingmagazine.com/2018/06/reference-guide-typography-mobile-web-design/
